### DISCUSSION TITLE
Feature: Don't stop batch process completely just because of one error.

### DISCUSSION THREAD

**Discussion by ReLater** [joomla/joomla-cms#43433] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
- Several com_content categories with 3000 articles or so exist.
- 75 articles or so should be moved from one category to another and get a tag.
- I had a long unordered paper list with ids.
- I opened the backend view com_content->articles and set the filters.
- Afterwards checked all ids in com_content->articles view. Scroll up, scroll down, click, scroll up, click and so on. Check what you have done and if it's correct. 
- Started a batch process: Move to category xy + Add tag yz.
- And get an message

`Batch process failed with following error: Another Article in this category has the same alias.`

- Some articles have been moved others not.

### Describe the solution you'd like
- The batch process doesn't stop the whole queue somewhere in betwwen (where?) and doesn't move some articles that don't have alias conflicts.
and/or
- An info which articles have the same alias. Which one haven't been moved.
and/or
- Move conflicting articles with a new alias (numbered one or easy to find one per filter?) and mention that.
and/or
- All articles have still their checks after page reload.

**Comment by chmst** [comment]
A similiar problem was when deleting a level from users access levels. https://github.com/joomla/joomla-cms/pull/43223
Do you tink that a similiar solution would help also for  batch prcessing articles and categories? Displaying a list of items which will cause an error and stop the process?

**Comment by rdeutz** [comment]
Please discuss what should be done, this here is not concreate enough. Moving to discussions meanwhile
